kaldb/src/main/java/com/slack/kaldb/server/ArmeriaService.java

```diff
@@ -102,6 +105,10 @@ private void addCompression(ServerBuilder sb) {
     sb.decorator(EncodingService.builder().newDecorator());
   }
 
+  private void setTimeout(ServerBuilder sb) {
```

Comment:
inline?

Reply:
I left this as a separate method because it matches what we did for `addCompression` and I believe we'll add additional timeout / exception handling. I went ahead and added a todo along with the mention about including a `Retry-After` based on our earlier Slack discussion, and referenced the api we should be able to use to achieve this.